quattor/aii/opennebula/default.pan

```diff
@@ -0,0 +1,73 @@
+unique template quattor/aii/opennebula/default;
+
+include 'quattor/aii/opennebula/schema';
+include 'quattor/aii/opennebula/functions';
+
+#  undef are set via schema default
+variable OPENNEBULA_AII_FORCE ?= undef; 
+variable OPENNEBULA_AII_ONHOLD ?= undef;
+variable OPENNEBULA_AII_FORCE_REMOVE ?= false;
+
+variable MAC_PREFIX ?= '02:00';
+
+"/system/aii/hooks/configure/" = {
+    append(nlist(
+        'module', OPENNEBULA_AII_MODULE_NAME,
+
+        "image", OPENNEBULA_AII_FORCE,
+        "template", OPENNEBULA_AII_FORCE,
+        ));
```

Comment:
Indentation here is weird, also please use `dict` throughout this code.

This also seems overly complicated considering you are appending a single item.

How about this?
```pan
'/system/aii/hooks/configure/' = append(dict(
    'module', OPENNEBULA_AII_MODULE_NAME,
    'image', OPENNEBULA_AII_FORCE,
    'template', OPENNEBULA_AII_FORCE,
));
```